guides/compatibility-mode.md

```diff
@@ -0,0 +1,32 @@
+# Compatibility Mode
+Compatibility mode allows `@angular/material` to work with [Angular Material for AngularJS](https://material.angularjs.org/latest/).
+
+## Upgrade Process Overview
+Below is a general overview of the upgrade process for an AngularJS application which uses [Angular Material for AngularJS](https://material.angularjs.org/latest/) and wishes to upgrade to Angular 2+ with `@angular/material`.
+1. Setup `@angular/upgrade/static` and bootstrap Angular in your AngularJS project
+2. Setup and enable compatibility mode for `@angular/material`
+3. Convert all components over to Angular and `@angular/material`
+4. Remove Angular Material for AngularJS
+5. Disable compatibility mode
+  - Find/replace `mat-` with `md-`
+  - Remove `NoConflictStyleCompatibilityMode`
+
+
+## Upgrade Checklist
+- [ ] Switch all `@angular/material` component prefixes in templates from `md-` to `mat-`
+  - See [this](https://github.com/angular/material2/blob/master/src/lib/core/compatibility/compatibility.ts#L33-L150) for a list of all components you need to rename.
```

Comment:
These line numbers won't be stable over time; I think just have an example here would be good